Makefile

```diff
@@ -186,6 +188,27 @@ xvfb:
 		sudo apt-get install --assume-yes --no-install-suggests xvfb; \
 	fi
 
+# symlinks the Go folder for the wpt.fyi project to (this) folder.
+wpt_fyi_symlink:
```

Comment:
Where's this recipe used?

Reply:
Can be used locally, for development setup.